modules/config-htpasswd-idp.adoc

```diff
@@ -2,24 +2,34 @@
 //
 // * identity_providers/config-identity-providers.adoc
 
+ifeval::["{context}" == "config-identity-providers"]
+:osd-distro:
+endif::[]
+ifeval::["{context}" == "rosa-sts-config-identity-providers"]
+:rosa-distro:
+endif::[]
+ifeval::["{context}" == "rosa-config-identity-providers"]
+:rosa-distro:
+endif::[]
+
 :_content-type: PROCEDURE
 [id="config-htpasswd-idp_{context}"]
 = Configuring an HTPasswd identity provider
 
 Configure an HTPasswd identity provider to create a single, static user with cluster administration privileges. You can log in to your cluster as the user to troubleshoot issues.
 
+[IMPORTANT]
+====
+The `htpasswd` identity provider option is included only to enable the creation of a single, static administration user. `htpasswd` is not supported as a general use identity provider for {product-title}.
```

Comment:
Just suggestion, I think `HTPasswd` is better than`htpasswd` in all sections for showing consistent term, not only here.

Reply:
@bysnupy thank you, that is a great suggestion. I have resolved this throughout the PR.